Apps/COMP.rc

```diff
@@ -0,0 +1,57 @@
+COMPONENT_TO_RECORD: NI
```

Comment:
No need to change this now, but I would like to find a better name in the long run.